src/shared/theme.tsx

```diff
@@ -0,0 +1,4 @@
+export const globalTheme: string | 'dark' | 'light' = 'dark';
+
+export const themeElement = (darkColor: string, lightColor: string) =>
+  globalTheme === 'dark' ? darkColor : lightColor;
```

Comment:
```suggestion
export type Theme = 'dark' | 'light';
export const globalTheme: Theme = 'dark';

export const themeElement = (darkColor: string, lightColor: string) =>
  globalTheme === ('dark' as Theme) ? darkColor : lightColor;
```
Still not going to allow `string` in there. This should compile.